web/src/message_edit.js

```diff
@@ -763,7 +764,13 @@ export function end_inline_topic_edit($row) {
 }
 
 export function end_message_row_edit($row) {
-    const message = message_lists.current.get(rows.id($row));
+    const row_id = rows.id($row);
+    const uppy = uppys[row_id];
+    // We need to cancel all uploads, reset their progress,
+    // and clear the files upon ending the edit.
+    uppy.cancelAll();
```

Comment:
This is a small thing, but I think something like `uppys[row_id].cancelAll()` would read clearer, or else name the variable something like `uploads_for_row`.

Reply:
I prefer `uploads_for_row.cancelAll();` for the sake of readability, even though `uppys[row_id].cancelAll()` saves us a line :)
Thanks for the feedback!